datm/datm_datamode_cfsr_mod.F90

```diff
@@ -191,14 +192,18 @@ subroutine datm_datamode_cfsr_advance(exportstate, mainproc, logunit, mpicom, ta
     lsize = size(strm_mask)
 
     if (first_time) then
+       call ESMF_VMGetCurrent(vm, rc=rc)
```

Comment:
@jedwards4b I am testing this under UFS and need to put all these code to the data mode just defined in their fork as a replacement of shr_mpi_max(). It might be nice to have a similar shared call, so everybody could use same code rather then copying all these ESMF staff to the data mode that needs to use shr_mpi_max(). Anyway, this is just and idea. Let me know what you think.

Reply:
I don't understand?  There are no calls to shr_mpi left anywhere in cdeps. Is ufs using it outside cdeps?

Reply:
@jedwards4b yes, UFS has some DATM modes that are not pushed to main repository. Anyway, I applied your changes and run `datm_cdeps_restart_cfsr` and `datm_cdeps_restart_cfsr` tests. They passed without any issue.